internal/node/node_repositories.bzl

```diff
@@ -198,11 +246,12 @@ def node_repositories(package_json):
 
   Args:
     package_json: a list of labels, which indicate the package.json files that need to be installed.
+    node_version: the version of NodeJS to install.
   """
   # Windows users need sh_binary wrapped as an .exe
   check_bazel_version("0.5.4")
 
-  _node_repo(name = "nodejs", package_json = package_json)
+  _node_repo(name = "nodejs", package_json = package_json, node_version = node_version)
 
   _yarn_repo(name = "yarn", package_json = package_json)
```

Comment:
do we need to give a way to control the yarn version too?

Reply:
Probably - but we can tackle that in a subsequent PR :)